scribejava-apis/src/test/java/com/github/scribejava/apis/examples/SalesforceAsyncExample.java

```diff
@@ -0,0 +1,123 @@
+package com.github.scribejava.apis.examples;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.net.URLEncoder;
+import java.util.Scanner;
+import java.util.concurrent.ExecutionException;
+
+import com.github.scribejava.apis.SalesforceApi;
+import com.github.scribejava.apis.salesforce.SalesforceToken;
+import com.github.scribejava.core.builder.ServiceBuilder;
+import com.github.scribejava.core.model.ForceTypeOfHttpRequest;
+import com.github.scribejava.core.model.OAuthAsyncRequestCallback;
+import com.github.scribejava.core.model.OAuthRequestAsync;
+import com.github.scribejava.core.model.Response;
+import com.github.scribejava.core.model.ScribeJavaConfig;
+import com.github.scribejava.core.model.Verb;
+import com.github.scribejava.core.oauth.OAuth20Service;
+import com.ning.http.client.AsyncHttpClientConfig;
+
+public abstract class SalesforceAsyncExample {
+
+    private static final String NETWORK_NAME = "Salesforce";
+
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public static void main(String[] args) throws InterruptedException, ExecutionException {
+        // Replace these with your client id and secret
+        final String clientId = "your client id";
+        final String clientSecret = "your client secret";
+
+        ScribeJavaConfig.setForceTypeOfHttpRequests(ForceTypeOfHttpRequest.FORCE_ASYNC_ONLY_HTTP_REQUESTS);
+        final AsyncHttpClientConfig clientConfig = new AsyncHttpClientConfig.Builder()
+                .setMaxConnections(5)
+                .setRequestTimeout(10_000)
+                .setAllowPoolingConnections(false)
+                .setPooledConnectionIdleTimeout(1_000)
+                .setReadTimeout(10_000)
+                .build();
+
+        final OAuth20Service service = new ServiceBuilder()
+                .apiKey(clientId)
+                .apiSecret(clientSecret)
+                .asyncHttpClientConfig(clientConfig)
```

Comment:
btw, there is no such method anymore in master
it becomes asyncNingHttpClientConfig